main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+var (
+	command string
+	resFile string
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		printUsage()
+	}
+
+	flag.Parse()
+
+	outFile, err := os.Create(resFile)
+	if err != nil {
+		quit(1, "Cannot open Output File\n")
```

Comment:
`quit(exitCode int, format string, args ...interface)`

Such as the `fmt` standard formatters: https://golang.org/pkg/fmt/#Printf

Reply:
this needs to be adressed...
right now quit() accepts a status code and an error (propagated from other functions)